Products/CMFPlone/_scripts/verifydb.py

```diff
@@ -43,20 +44,28 @@ def verify_zodb(obj, debug=False):
     next_ = None
     count = 0
     errors = 0
+    issues = []
     while True:
         count += 1
         oid, tid, data, next_ = storage.record_iternext(next_)
         logger.debug('Verifying {}'.format(oid))
-        success = verify_record(oid, data, debug)
+        success, msg = verify_record(oid, data, debug)
         if not success:
             errors += 1
+            issues.append(msg)
         if next_ is None:
             break
 
+    issues = dict(Counter(sorted(issues)))
```

Comment:
Why do you convert to a dict? `Counter` extends dict and supports the `items` method.

Reply:
good point